network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/subscription/NetworkProtectionAccessStateImpl.kt

```diff
@@ -39,11 +37,10 @@ class NetworkProtectionAccessStateImpl @Inject constructor(
     private val networkProtectionState: NetworkProtectionState,
     private val dispatcherProvider: DispatcherProvider,
     private val netpSubscriptionManager: NetpSubscriptionManager,
-    private val subscriptions: Subscriptions,
 ) : NetworkProtectionAccessState {
 
     override suspend fun getState(): NetPAccessState = withContext(dispatcherProvider.io()) {
-        if (isTreated()) {
+        if (true) {
```

Comment:
Why keep this?

Reply:
argh, I refactored that step by step and forgot to let the IDE simplify that. Done